src/utils/scripts.ts

```diff
@@ -164,7 +164,14 @@ export async function runScript(
     }
 
     for (const script of scripts) {
-        await execaCommand(script, { cwd, shell: true, env: environment });
+        await execaCommand(script, {
+            cwd,
+            shell: true,
+            env: {
+                ...process.env,
```

Comment:
I think this is automatically inherited https://github.com/sindresorhus/execa/blob/main/docs/api.md#optionsextendenv.

> If true, the subprocess uses both the [env](https://github.com/sindresorhus/execa/blob/main/docs/api.md#optionsenv) option and the current process' environment variables ([process.env](https://nodejs.org/api/process.html#processenv)). If false, only the env option is used, not process.env.